Tuist/Package.swift

```diff
@@ -29,14 +29,6 @@ let package = Package(
             url: "https://github.com/pointfreeco/swift-snapshot-testing",
             revision: "26ed3a2b4a2df47917ca9b790a57f91285b923fb"
         ),
-        .package(
-            url: "https://github.com/RevenueCat/purchases-ios",
-            branch: "main"
```

Comment:
Tuist/ProjectDescriptionHelpers/ProjectDescriptionPackages.swift already handles local vs remote packages based on dependencyMode, so this dependency is redundant